Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -415,7 +418,7 @@ void qMRMLSubjectHierarchyTreeView::setCurrentItems(QList<vtkIdType> items)
     return;
     }
 
-  this->selectionModel()->clearSelection();
+  this->clearSelection();
```

Comment:
Can you please add a comment why calling this is necessary?

Reply:
I do not know what motivates the clear before the change of items, this was here prior to my changes, I just replaced the `model::clearSelection()` by the wrapped version of that function in that class.

Reply:
Oh, OK then. Thanks for the explanation! I just thought based on the GitHub diff as I saw it that this is a newly added line, and in that case there must have been a reason for adding it. As it's not a new line, I don't expect you to figure out the reason.